### PR TITLE
100% classes

### PR DESCRIPTION
**Pull request by skyrising**

### CODE REVIEW

**Comment by hYdos** [comment]
1.12.2 progress is flying

**Comment by halotroop2288** [comment]
Petition to re-run all matches starting from 1.12.2 when this is over, cause that would probably put us a lot further along.

File: mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping [hunk 1]
```diff
@@ -1,6 +1,5 @@
 CLASS net/minecraft/class_3856 net/minecraft/client/render/entity/feature/FeatureRenderer
 	METHOD method_31187 render (Lnet/minecraft/class_2267;FFFFFFF)V
-		ARG 5 limbAngle
```
Comment (halotroop2288) [inline mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping:3]: Not easy to see the logic behind these changes from Github without having Enigma open, but this is a strange one. I'm interested how you came up with this one.

File: mappings/net/minecraft/client/render/item/BannerRenderers.mapping [hunk 2]
```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/class_5876 net/minecraft/client/render/item/BannerRenderers
```
Comment (halotroop2288) [inline mappings/net/minecraft/client/render/item/BannerRenderers.mapping:1]: `BannerRenderer`*?

**Review by halotroop2288 — COMMENTED** [review]
Good job, all-in-all. This is a big step towards fully complete mappings.
I'd say it's worthy of setting a milestone for it: "100% Complete Classes"

File: mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping (hunk 1 above)
Comment (skyrising) [reply mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping:3]: At least args 6 & 7 were incorrect and I'm pretty sure this is as well, but I didn't spend time investigating what the proper name is

Comment (halotroop2288) [reply mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping:3]: I'll look at it later if I remember. I usually search around until I find an assignment in a child or super class, even looking at modern sometimes. Understandable that you wouldn't want to go through that much trouble for some args.

File: mappings/net/minecraft/client/render/item/BannerRenderers.mapping (hunk 2 above)
Comment (skyrising) [reply mappings/net/minecraft/client/render/item/BannerRenderers.mapping:1]: The point of this class is to provide 2 fields implementing `BannerRenderers.Renderer`

Comment (halotroop2288) [reply mappings/net/minecraft/client/render/item/BannerRenderers.mapping:1]: I think there are still better names for it, but I guess we can leave that up in the air for now.

**Review by halotroop2288 — APPROVED** [review]

**Review by BluCobalt — APPROVED** [review]
Ok this is pretty epic

**Review by BoogieMonster1O1 — APPROVED** [review]

**Pull request merged**